client/public/_redirect

```diff
@@ -0,0 +1 @@
+/*    /index.html   200
```

Comment:
This file is added because all routes (client-side), other than /, are not available when deployed on netlify. This is just to ensure that all routes are served from index.html